modules/infra/src/main/scala/scaladex/infra/elasticsearch/ElasticsearchMapping.scala

```diff
@@ -74,6 +74,7 @@ object ElasticsearchMapping {
     objectField("formerReferences").fields(
       textField("organization").analyzer("standard"),
       textField("repository").analyzer("standard")
-    )
+    ),
+    nestedField("githubInfo.commitActivity")
```

Comment:
Now it is `commitsPerYear: Option[Int]`
```suggestion
    intField("githubInfo.commitsPerYear")
```